src/ui/pages/SwapForm/SwapForm.tsx

```diff
@@ -216,9 +217,14 @@ export function SwapFormComponent() {
     client,
     positions,
     asset_code: null,
-    getNativeAsset: ({ chain }) => getNativeAsset({ chain, currency }),
+    getNativeAssetId: async (chain) => {
+      const networksStore = await getNetworksStore();
+      const networks = await networksStore.load();
```

Comment:
in getNativeAsset, we used to do `await networksStore.load({ chains: [chain.toString()] });`. Why isn't it required anymore? Or should we still pass this param?